manifest-agent.tmpl

```diff
@@ -0,0 +1,23 @@
+image: rancher/fleet-agent:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}
+manifests:
+  -
+    image: rancher/fleet-agent:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}-amd64
+    platform:
+      architecture: amd64
+      os: linux
+  -
+    image: rancher/fleet-agent:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}-arm
+    platform:
+      architecture: arm
+      os: linux
+  -
+    image: rancher/fleet-agent:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}-arm64
+    platform:
+      architecture: arm64
+      os: linux
+  -
+    image: rancher/fleet-agent:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}-windows-amd64-1909
```

Comment:
you can add 2004 and 20H2 if youre motivated, you already have v0.5.4 dapper and those two are not available there. just copy pasta and be done with it now.

Reply:
Done